templates/bot/js/default/package.json

```diff
@@ -9,6 +9,8 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
+    "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
```

Comment:
Will teams toolkit run this script for local debug?

Reply:
Yes, first we will adjust `tasks.json` to let vscode run this script when F5, next we will adjust teamsfx CLI to run this script when preview.